traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/monitor/TrafficMonitorWatcher.java

```
@@ -277,10 +274,18 @@ public void processConfig() {
 		try {
 			String hostList = System.getenv("TRAFFIC_MONITOR_HOSTS");
 
-			final File trafficMonitorConfigFile = new File(propertiesDirectory, monitorProperties);
+			File trafficMonitorConfigFile;
+
+			if (monitorProperties.matches("^\\w+:.*")) {
```

I specifically tried that, but the exception was not handled/caught where we expected.